Limelight/Unconfirmed 783656.crdownload

```diff
@@ -0,0 +1 @@
+{"area_max":100.0,"area_min":0.001,"area_similarity":0.0,"aspect_max":20.0,"aspect_min":0.0,"barcode_type":"qrzx","black_level":0,"blue_balance":1975.0,"botfloorsnap":0,"botlength":0.7112,"bottype":"swerve","botwidth":0.7112,"calibration_type":0,"classifier_conf":0.1,"classifier_runtime":"cpu","clip_labels":"face,hand,computer,keyboard","contour_grouping":0,"contour_sort_final":0,"convexity_max":100.0,"convexity_min":10.0,"corner_approx":5.0,"crop_focus":0.0,"crop_x_max":1.0,"crop_x_min":-1.0,"crop_y_max":1.0,"crop_y_min":-1.0,"cross_a_a":1.0,"cross_a_x":0.0,"cross_a_y":0.0,"cross_b_a":1.0,"cross_b_x":0.0,"cross_b_y":0.0,"debugpipe":0,"depthrange_max":100.0,"depthrange_min":0.0,"desc":"Pipeline_Right","desired_contour_region":0,"detector_conf":0.8,"detector_idfilters":"","detector_runtime":"cpu","dilation_steps":0,"direction_filter":0,"dual_close_sort_origin":0,"erosion_steps":0,"exposure":30.0,"fiducial_backend":"umich","fiducial_denoise":0.0,"fiducial_idfilters":"","fiducial_locfilters":"","fiducial_qualitythreshold":2.0,"fiducial_resdiv":2,"fiducial_size":165.1,"fiducial_skip3d":0,"fiducial_type":"aprilClassic36h11","fiducial_vis_mode":"3dbotposefieldspace","flicker":0,"force_convex":1,"hue_max":85,"hue_min":55,"image_flip":0,"image_source":0,"img_to_show":0,"intersection_filter":0,"invert_hue":0,"lcgain":6.0,"margin_tv":0.2,"multigroup_max":7,"multigroup_min":1,"multigroup_rejector":0,"nnp_rotate":0,"pipeline_led_enabled":1,"pipeline_led_power":100,"pipeline_res":0,"pipeline_type":"pipe_fiducial","python_snapscript_name":"","quality_focus":0.3,"red_balance":1200.0,"reverse_morpho":0,"roi_x":0.0,"roi_y":0.0,"rsf":-0.1905,"rspitch":18.0,"rsroll":0.0,"rss":0.273,"rsu":0.3048,"rsyaw":0.0,"sat_max":255,"sat_min":70,"send_corners":0,"send_json":0,"tsf":0.0,"tss":0.0,"tsu":0.0,"tv_conf":0.5,"val_max":255,"val_min":70,"x_outlier_miqr":1.5,"y_outlier_miqr":1.5,"yaw_latency_adjustment":0.0,"zsclassifier_conf":0.1}
```

Comment:
Oh I guess we didn't confirm the downloads so these didn't get the right names. We can fix these tomorrow.